scripts/run_presubmit_checks.py

```diff
@@ -52,24 +55,28 @@
     help='optional; if specified, the origin branch to compare against.')
 
 
-def main(args=None) -> None:
+def main(args: Optional[List[str]] = None) -> None:
     """Run the presubmit checks."""
     parsed_args = _PARSER.parse_args(args=args)
 
     # Run Javascript and Python linters.
     print('Linting files since the last commit')
-    pre_commit_linter.main(args=[])
+    pre_commit_linter.main(args=[])  # type: ignore[no-untyped-call]
     print('Linting passed.')
     print('')
 
-    current_branch = subprocess.check_output([
-        'git', 'rev-parse', '--abbrev-ref', 'HEAD'])
+    current_branch = subprocess.check_output(
+        ['git', 'rev-parse', '--abbrev-ref', 'HEAD'], text=True
+    )
 
     # If the current branch exists on remote origin, matched_branch_num=1
     # else matched_branch_num=0.
-    matched_branch_num = subprocess.check_output([
-        'git', 'ls-remote', '--heads', 'origin', current_branch, '|', 'wc',
-        '-l'])
+    matched_branch_num = subprocess.check_output(
+        [
+        'git', 'ls-remote', '--heads', 'origin', current_branch, '|', 'wc', '-l'
```

Comment:
Indentation is off. Previous indentation was fine, but if you want it like this then this line must be indented by 4.

Reply:
> Indentation is off 

Fixed